online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingConfiguration.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 23/8/2022.
+ */
+
+package com.adyen.checkout.onlinebankingcz
+
+import android.content.Context
+import android.os.Parcel
+import android.os.Parcelable
+import com.adyen.checkout.core.api.Environment
+import com.adyen.checkout.issuerlist.IssuerListConfiguration
+import java.util.Locale
+
+class OnlineBankingConfiguration : IssuerListConfiguration {
```

Comment:
This class should be called `OnlineBankingCZConfiguration` right? Same for the delegate, input data, output data, and model.

Reply:
I'll redo this after making `online-banking-core` module in the next PR